hurumap/templates/settings_js.html

```diff
@@ -1,8 +1,11 @@
 {% extends 'settings_js.html' %}
+{% load jsonify %}
 
 {% block settings_javascript %}
 {{block.super}}
 
 var SITE_TAGLINE = '{{ WAZIMAP.title_tagline }}';
 var SITE_DESCRIPTION = '{{ WAZIMAP.description }}';
+WAZIMAP_COUNTRY_CODE = '{{ HURUMAP.country_code }}';
```

Comment:
why not `SITE_COUNTRY_CODE`?

Reply:
this is a variable used by wazimap's `widget.geo.select.js`

```
var textmatchAPI = '/place-search/json/',
    geocodingAPI = 'https://maps.googleapis.com/maps/api/geocode/json?address=%QUERY&components=country:' + WAZIMAP_COUNTRY_CODE + '&region=' + WAZIMAP_COUNTRY_CODE + '&key=' + GOOGLE_GEOCODE_API_KEY,
    resultTemplate = Handlebars.compile('<p class="result-name"><span class="result-type">{{geo_level}}</span>{{full_name}}</p>'),
    geoSelect = $('#geography-select, #geography-select-home');
```

Reply:
But why are we then setting it here? Shouldn't it come directly from `{{block.super}}`?